package.json

```diff
@@ -18,6 +18,7 @@
   "engines": {
     "node": ">= 10.13.0"
   },
+  "packageManager": "npm@6.14.8",
```

Comment:
@valscion 

By default `npm` is already shipped with Node.js, so you don't need to enable `corepack` to use `npm`, that why the docs said `no use`. But, the field still helps when a specific version of npm is needed.

I have been using the latest version of npm (which uses the lockfile v2/v3) for a while. However, the `webpack-bundle-analyzer` still uses the v1 version of `package-lock.json` and I need to use an older version of npm. With this field it does help me enable the specific version of npm.

Reply:
Okay cool! I didn't know that this allowed corepack to use an older version of npm. This is useful!

Reply:
based on https://nodejs.org/api/corepack.html#how-does-corepack-interact-with-npm, I don't think corepack will install an older npm version

Reply:
> based on https://nodejs.org/api/corepack.html#how-does-corepack-interact-with-npm, I don't think corepack will install an older npm version

But for some reason, it does work for me though. I am able to use the 6.x version of npm to generate the correct `package-lock.json` that won't bloat the change.

Reply:
@stof I might have found out the reason that `corepack` does work for me. It has something to do with my special setup:

- I use `fnm`, a tool like `nvm`, to manage my Node.js versions
- Though not recommended, I've manually specified `NPM_CONFIG_PREFIX` to let npm install the global packages to my specific location (instead of the default location)
- I've upgraded my npm version with `npm i npm -g`, so my npm is installed in `NPM_CONFIG_PREFIX` instead of the default location.
- In my `$PATH`, I add the Node.js path before the `NPM_CONFIG_PREFIX` path, so the shell will choose npm from `corepack` when available.

<img width="944" alt="image" src="https://github.com/webpack-contrib/webpack-bundle-analyzer/assets/40715044/29de9212-7b05-4689-8479-39162ce1bc5f">

Reply:
Okay, so a special setup is the reason.

I don't think having the `packageManager` line hurts in any way, though. Or do you think it's a bad idea to have in the repository?

Reply:
> Okay, so a special setup is the reason.
> 
> I don't think having the `packageManager` line hurts in any way, though. Or do you think it's a bad idea to have in the repository?

Yes, it doesn't hurt.

This `packageManager` field has no effect on the package's users. Maintainers/contributors who don't have corepack won't be affected either. Maintainers/contributors who have enabled corepack **without a specific setup** will have an extra copy of corepack's `npm` installed, it just won't be chosen by the shell (The shell will prefer the original npm without the setup).